config/app_queue.php

```diff
@@ -36,5 +36,8 @@
 
 		// set default Mailer class
 		'mailerClass' => 'Cake\Mailer\Email',
+
+		// set default datasource connection
+		'datasourceconnection' => 'default',
```

Comment:
just connection maybe?
or why datasourceconnection?

Reply:
default it to null here please.

Reply:
I've used 'datasourceconnection' because I think that 'connection' is too ambiguous. 
I've initially planned to use 'datasource' (as in cake2), but in cake3 it's named Connection, that's why I have finally wrote 'datasourceconnection'.

If you think that 'connection' is better and more correct, I do so

Reply:
why do you need this btw?

Reply:
then use datasourceConmection maybe